init/common.js

```diff
@@ -5,6 +5,7 @@ const _ = require("lodash");
 const authenticatedRolePermissions = require("./permissions/authenticated");
 const publicRolePermissions = require("./permissions/public");
 const config = require("./config");
+const { default: init } = require("@strapi/strapi/dist/Strapi");
```

Comment:
Please don't import from distribution folders. You are not using this import

Reply:
@nikita-pardeshi-github should we remove this then?